care/facility/api/viewsets/patient.py

```diff
@@ -649,8 +651,34 @@ def perform_create(self, serializer):
             raise ValidationError(
                 {"patient": "Only active patients data can be updated"}
             )
-        return serializer.save(
+
+        instance = serializer.save(
             facility=patient.facility,
             patient=patient,
             created_by=self.request.user,
         )
+
+        message = {
+            "facility_id": str(patient.facility.external_id),
+            "patient_id": str(patient.external_id),
+            "from": "patient/doctor_notes/create",
+        }
+
+        NotificationGenerator(
```

Comment:
Do we really need 2 notifications for the same thing?

Reply:
I believe so, we have a clear distinction between a Notification and a standalone webpush message in the frontend, the webpush hook we have does not respond if it's just a notification. We could either make it capture all Notifications but I feel that might cause some side effects since till now it only captures webpush "messages" and notification types are not sent to it.
So that's why two NotificationGenerator, one is for the actual Browser notification that the patient note was added and one is just a one time webpush message captured by webpush hook we have to refetch the patient notes. (This message is one off)